Dockerfile

```diff
@@ -1,17 +1,20 @@
-ARG CP_KAFKA_CONNECT_BASE_VERSION=6.0.1
+ARG CP_KAFKA_CONNECT_BASE_VERSION=7.2.2
 
 FROM confluentinc/cp-kafka-connect-base:$CP_KAFKA_CONNECT_BASE_VERSION
 
 ENV COMPONENT=kafka-connect
+ENV CONNECT_PLUGIN_PATH=/usr/share/java,/usr/share/confluent-hub-components,/usr/share/filestream-connectors
```

Comment:
Do we need the filestream-connectors plugin path here if we have to specify it anyway in the KafkaConnect CR?

Reply:
Im going to remove this